### PR TITLE
partitioned-table: adding the correction to incorrect examples

### PR DESCRIPTION
**Pull request by hawkingrei**
Signed-off-by: Weizhen Wang <wangweizhen@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

adding the correction to incorrect examples. it's helpful for reader to understand it.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: partitioned-table.md [hunk 1]
```diff
@@ -1018,7 +1018,23 @@ PARTITION BY HASH(col1 + col3)
 ERROR 1491 (HY000): A PRIMARY KEY must include all columns in the table's partitioning function
 ```
 
-原因是 `col1` 和 `col3` 出现在分区键中，但是几个唯一键定义并没有完全包含它们。
+原因是 `col1` 和 `col3` 出现在分区键中，但是几个唯一键定义并没有完全包含它们， 需做如下修改即为合法语句:
```
Comment (djshow832) [inline partitioned-table.md:1021]:
```suggestion
原因是 `col1` 和 `col3` 出现在分区键中，但是几个唯一键定义并没有完全包含它们，需做如下修改即为合法语句:
```

Comment (AilinKid) [inline partitioned-table.md:1021]:
```suggestion
原因是 `col1` 和 `col3` 出现在分区键中，但是几个唯一键定义并没有完全包含它们， 需做如下修改为合法语句:
```

Comment (AilinKid) [reply partitioned-table.md:1021]: below is the same

**Review by djshow832 — COMMENTED** [review]
Rest LGTM.

File: partitioned-table.md (hunk 1 above)
Comment (djshow832) [reply partitioned-table.md:1021]: I don't think it needs to be modified.

Comment (hawkingrei) [reply partitioned-table.md]: add some description

**Review by hawkingrei — COMMENTED** [review]
Delete extra spaces.

**Review by AilinKid — APPROVED** [review]

**Review by djshow832 — APPROVED** [review]

File: partitioned-table.md [hunk 2]
```diff
@@ -1064,7 +1080,34 @@ PARTITION BY HASH( YEAR(col2) )
 PARTITIONS 4;
 ```
 
-两个例子中，主键都没有包含分区表达式中的全部的列。
+两个例子中，主键都没有包含分区表达式中的全部的列，在主键中补充缺失列即为合法语句:
```
Comment (TomShawn) [inline partitioned-table.md:1083]:
```suggestion
以上两个例子中，主键都没有包含分区表达式中的全部的列，在主键中补充缺失列后语句即为合法:
```

File: partitioned-table.md [hunk 3]
```diff
@@ -1018,7 +1018,23 @@ PARTITION BY HASH(col1 + col3)
 ERROR 1491 (HY000): A PRIMARY KEY must include all columns in the table's partitioning function
 ```
 
-原因是 `col1` 和 `col3` 出现在分区键中，但是几个唯一键定义并没有完全包含它们。
+原因是 `col1` 和 `col3` 出现在分区键中，但是几个唯一键定义并没有完全包含它们，需做如下修改即为合法语句:
```
Comment (TomShawn) [inline partitioned-table.md:1021]:
```suggestion
原因是 `col1` 和 `col3` 出现在分区键中，但是几个唯一键定义并没有完全包含它们，需做如下修改语句即为合法:
```

File: partitioned-table.md (hunk 2 above)
Comment (hawkingrei) [reply partitioned-table.md]: Done

File: partitioned-table.md (hunk 3 above)
Comment (hawkingrei) [reply partitioned-table.md]: Done

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**